### PR TITLE
[DAT-4086] Liquibase: Implement New Landing Page && [DAT-4021] - Liquibase: Update Header and Footer for the Key Sites

### PR DESCRIPTION
**Pull request by tabuckner**
## What I Did
Added 3p redesign to existing site architecture. 

These files currently live under `/redesign`. **We are not using the node based build workflow currently** The tooling was likely set up by the developer for ease of development and does not _need_ to replace _or integrate_ with the existing setup at this time.

I've also refactored our tracking codes a bit for ease of integration with the 3p vendors static assets.

Additionally I also moved the favicons to a partials directory under includes. 

NOTE: We dont _HAVE_ to track the 3p vendor's source files, although it would definitely make altering the designs much easier in the future. For now I propose we add them to SCM to make our lives easier.

## How To Test
Currently can be accessed at:
https://daticalteam.slack.com/archives/CM8GN44SD/p1581106886023000

This will be up as long as my laptop doesn't go to sleep. 

Get in there and take a look before that happens. 

On Monday morning I will open it back up.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,575 @@
+---
```
Comment (SteveDonie) [inline index.html]: I note that this is not using the  _includes/components/redesign/footer.html and  _includes/components/redesign/header.html

File: redesign/assets/img/debug.log [hunk 2]
```diff
@@ -0,0 +1,228 @@
+[0204/001353.106:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
Comment (SteveDonie) [inline redesign/assets/img/debug.log]: This file should not be in git

Comment (tabuckner) [reply redesign/assets/img/debug.log]: OOH good catch. thanks!!! @SteveDonie

**Comment by tabuckner** [comment]
> Looks mostly nice. The existing pages are using the new header and footer, and that does not look good.
> 
> I have committed a change that removes all the absolute URLs referring to `https://www.liquibase.org` so that when using the site in local mode, the links don't send you off to the live site.

MY MAN! thank you :D

File: redesign/assets/img/debug.log (hunk 2 above)
Comment (SteveDonie) [reply redesign/assets/img/debug.log]: removed.

**Pull request merged**